tool/python/singa/model.py

```diff
@@ -325,7 +325,7 @@ def set_cudnn_layer_type(self, net):
             elif ly_type == kSoftmaxLoss: cudnn_ly_type = kCudnnSoftmaxLoss
             elif ly_type == kSTanh:
                 cudnn_ly_type = kCudnnActivation
-                net.layer[i].activation_conf.type = STANH
+                net.layer[i].activation_conf.type = TANH
```

Comment:
STANH is not available in cudNN.
Hence if users configure stanh, then just report error?